### DISCUSSION TITLE
handbook/mobile-testing/

### DISCUSSION THREAD

**Discussion by giscus** [serenity-js/serenity-js#1624] [Comments]
# handbook/mobile-testing/

Serenity/JS integrates with WebdriverIO to enable mobile test automation using the Appium protocol.

https://serenity-js.org/handbook/mobile-testing/

**Comment by shoumik121** [comment]
interested

**Comment by tibioneh** [comment]
Hello, when will this be available? At the moment there doesn't seem to be a way to use SerenityJS with WebdriverIO and Appium. Thank you.

**Reply by jan-molak** [reply]
Hi there and thanks for your interest in Serenity/JS! Please follow https://github.com/serenity-js/serenity-js/issues/1279 to stay up to date with the progress and get involved with the development of this feature
